# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/cairo-quartz.h
# Commit: 61ef06598b0c
# Full Hash: 61ef06598b0c87016b6eecabf22cde2b7f1fbeb6
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-05-07 09:13:07
# Regressor Bug: 1892913
# File Overlap Count: 2
# Description:
#   Bug 1892913 - patch 18 - Cairo fixes for iOS build. r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D209426
# ==============================================================================

diff -r 51b400f880e3 -r 61ef06598b0c gfx/cairo/cairo/src/cairo-quartz.h
--- a/gfx/cairo/cairo/src/cairo-quartz.h	Mon May 06 19:25:20 2024 +0000
+++ b/gfx/cairo/cairo/src/cairo-quartz.h	Mon May 06 19:25:20 2024 +0000
@@ -49,6 +49,10 @@
 #include <ApplicationServices/ApplicationServices.h>
 #endif
 
+#if CAIRO_HAS_IMAGE_IO
+#include <ImageIO/ImageIO.h>
+#endif
+
 CAIRO_BEGIN_DECLS
 
 cairo_public cairo_surface_t *
